crates/red_knot_test/src/lib.rs

```diff
@@ -28,7 +28,7 @@ const MDTEST_TEST_FILTER: &str = "MDTEST_TEST_FILTER";
 #[allow(clippy::print_stdout)]
 pub fn run(path: &Utf8Path, long_title: &str, short_title: &str, test_name: &str) {
     let source = std::fs::read_to_string(path).unwrap();
-    let suite = match test_parser::parse(short_title, &source) {
+    let suite = match test_parser::parse(short_title, &source, path) {
         Ok(suite) => suite,
         Err(err) => {
             panic!("Error parsing `{path}`: {err:?}")
```

Comment:
Here's where we already show the path in every error.

Reply:
Looks good, to keep this PR focused. Future PR to further improve error output here is welcome!